reference/5-api-publisher/Considerations-docker-configuration-and-execution.md

```diff
@@ -0,0 +1,171 @@
+# Running in Docker Desktop.
```

Comment:
Please remove the period at the end of the heading.

The file name and title should be the same, so either change the title to "Considerations for Docker Configuration and Execution" or change the file to `running-in-docker-desktop.md`.

Reply:
fixed